lib/renderer.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const less = require('less');
+const { basename, dirname, join} = require('path');
+
+module.exports = async function(data) {
+  this.theme.config.less = Object.assign({
+    paths: []
+  }, this.theme.config.less);
```

Comment:
usually this is implemented in `index.js`, but somehow it doesn't work for theme config, so I put here as a workaround.